json-schemas/bulkDownload.json

```diff
@@ -0,0 +1,99 @@
+{
+  "type": "object",
+  "properties": {
+      "id": {
+          "type": "integer"
+      },
+      "params_json": {
+          "type": "string"
+      },
+      "download_type": {
+          "type": "string"
+      },
+      "status": {
+          "type": "string"
+      },
+      "error_message": {
+          "type": "NoneType"
```

Comment:
whats the purpose of these `NoneType`s?

Reply:
Those are most often errors from the compiler thanks for pointing it out!